src/components/clusterDetail/ClusterCredentials.tsx

```diff
@@ -68,11 +68,7 @@ const ClusterCredentials: React.FC<ClusterCredentialsProps> = ({
     );
   }
 
-  return (
-    <GridItem span={12} lg={10} xl={6}>
-      {credentialsBody}
-    </GridItem>
-  );
+  return <GridItem span={12}>{credentialsBody}</GridItem>;
```

Comment:
This makes the password clipboard copy 100% width which is too much
<img width="2048" alt="Screenshot 2021-04-14 at 12 59 45" src="https://user-images.githubusercontent.com/1121740/114700183-7bb6ef00-9d21-11eb-83d6-1cf665a89696.png">